src/card.css

```diff
@@ -0,0 +1,95 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__imac {
+  width: 160px;
+  height: 134px;
+  margin-top: 32px;
+  margin-bottom: 40px;
+  margin-left: 19px;
+  margin-right: 19px;
```

Comment:
do not add horizontal margins for every element, just apply padding on the whole card